dom.bs

```diff
@@ -1204,6 +1220,9 @@ for discussion).
  <li><p>Append (<var>target</var>, <var>targetOverride</var>, <var>relatedTarget</var>) to <var>event</var>'s
  <a for=Event>path</a>.
 
+ <li><p>Let <var>activationTarget</var> be <var>target</var>, if <var>target</var> has
```

Comment:
Hmm, this is a pretty common pattern. I guess I can start removing commas whenever I see them, but it does make it a bit more readable I think.